doc/source/drivers/vector/csv.rst

```diff
@@ -413,6 +413,15 @@ The following open options are supported:
 
       Maximum number of bytes for a line (-1=unlimited).
 
+-  .. oo:: OGR_SCHEMA
+      :choices: <filename>|<json string>
+      :since: 3.11.0
+
+      Partially or totally overrides the auto-detected schema to use for creating the layer.
```

Comment:
We could mention this option has precedence over the AUTODETECT_TYPE open option and the .csvt side car file.

Reply:
done